cli/central-client/src/main/java/org/ballerinalang/central/client/Utils.java

```diff
@@ -419,9 +430,46 @@ private static String getPlatformFromBala(String balaName, String packageName, S
         return balaName.split(packageName + "-")[1].split("-" + version)[0];
     }
 
-    private static void extractBala(Path balaFilePath, Path balaFileDestPath) throws IOException {
+    private static byte[] checkHash(String filePath, String algorithm) {
```

Comment:
Shall we move the `checkHash` method below the `extractBala`? The other methods seem to be following the pattern of the caller method being above the callee.

Reply:
Fix in https://github.com/ballerina-platform/ballerina-lang/pull/41490/commits/8502a363b75e58a56c9163ad14036facc8bdd74e